functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -11,34 +11,24 @@ First Responder Kit links:
 http://FirstResponderKit.org
 https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit
 
-
 .PARAMETER SqlInstance
 SQL Server instance or collection of SQL Server instances
 
 .PARAMETER Database
-Database to store the FRK stored procs, typically master
+Database to store the FRK stored procs, typically master and master by default
 
 .PARAMETER SqlCredential
 Use SqlCredential to connect to SqlInstance with SQL authentication. 
 If SqlCredential is not specified, Windows authentication will be used.
 
+.PARAMETER Silent
+Use this switch to disable any kind of verbose messages
+
 .NOTES 
 Original author: Tara Kizer, Brent Ozar Unlimited (https://www.brentozar.com/)
-dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
-Copyright (C) 2016 Chrissy LeMaire
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Website: https://dbatools.io
+Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
```

Comment:
The shorter one can be found on the updated templates at https://github.com/sqlcollaborative/dbatools-templates